core/src/main/ruby/jruby/truffle/core/rubinius/kernel/bootstrap/time.rb

```diff
@@ -25,11 +25,10 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 class Time
-  # TODO(CS)
-  #def self.now
-  #  Rubinius.primitive :time_s_now
-  #  raise PrimitiveFailure, "Time.now primitive failed"
-  #end
+  def self.now
+    Rubinius.primitive :time_s_now
```

Comment:
Primitive call here. Note the unusual semantics - fall through for cases not handled.